integration_tests/integration_test.py

```diff
@@ -0,0 +1,131 @@
+"""
+This script is used to test the integration of the mitx-graders library with the Open edX platform.
+Running a code that uses the functions provided by the library using the safe_exec function, and in the MIT course context,
+to be able to use the python_lib.zip that contains the library.
+"""
+
+import os
+import django
+import logging
+from xmodule.capa.safe_exec import safe_exec
+from xmodule.util.sandboxing import SandboxService
+from xmodule.contentstore.django import contentstore
+from opaque_keys.edx.keys import CourseKey
+
+# Set up Django environment
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "cms.envs.test")
+django.setup()
+
+log = logging.getLogger(__name__)
+
+# Define the code to be executed
+all_code = """
```

Comment:
```suggestion
GRADING_CLASSES_CODE = """
```

Reply:
What would be the easiest way to maintain this?

Reply:
We can use something like the following:
```python
from mitxgraders import *

StringGrader()
FormulaGrader()
NumericalGrader()
MatrixGrader()
.
.
.
```
or
```python
from mitxgraders import (
    StringGrader,
    FormulaGrader,
    NumericalGrader,
    MatrixGrader
)
```
That could test that we can use the Grader functions we want.

I added more context to the code to cover a little more in the tests, using some examples from the unit tests. However, this may not be necessary since it has already been tested in the unit tests. We can test only the fact that we can use the graders. I think this could be totally up to @blarghmatey.

@blarghmatey, do you prefer a code with some examples but with more maintenance or only a test that we can use the graders, which is the main functionality, with less maintenance,  and leave the rest to the unit test?

Thanks, @mariajgrimaldi, for that observation.